source/patterns/@aws-solutions-constructs/core/test/glue-job-helper.test.ts

```diff
@@ -32,11 +32,15 @@ test('Test deployment with role creation', () => {
     assumedBy: new ServicePrincipal('glue.amazonaws.com')
   });
 
-  const cfnJobProps: CfnJobProps = defaults.DefaultGlueJobProps(_jobRole, {
-    name: _jobID,
-    pythonVersion: '3',
-    scriptLocation: 's3://fakelocation/script'
-  }, 'testETLJob', {}, '1.0');
+  const _userProvidedCfnJobProps: Partial<CfnJobProps> = {
```

Comment:
The Solutions Constructs team is _considering_ moving to this to replace `| any`. We shouldn't be rolling it out all through the library at this point.